src/test/java/org/sagebionetworks/bridge/exporter/util/BridgeExporterUtilTest.java

```diff
@@ -202,4 +222,132 @@ public void shouldConvertFreeformTextToAttachment() {
         assertFalse(BridgeExporterUtil.shouldConvertFreeformTextToAttachment(other, "other"));
     }
+
+    @Test
+    public void canConvertToColumnList() {
+        final String TEST_COLUMN_MODEL_NAME_1 = "test_column_model_1";
+        final String TEST_COLUMN_MODEL_NAME_2 = "test_column_model_2";
+
+        List<ColumnModel> TEST_COLUMN_LIST;
+
+        ImmutableList.Builder<ColumnModel> columnListBuilder = ImmutableList.builder();
+
+        ColumnModel TEST_MODEL_1 = new ColumnModel();
+        TEST_MODEL_1.setName(TEST_COLUMN_MODEL_NAME_1);
+        TEST_MODEL_1.setColumnType(ColumnType.STRING);
+        TEST_MODEL_1.setMaximumSize(36L);
+        columnListBuilder.add(TEST_MODEL_1);
+
+        ColumnModel TEST_MODEL_2 = new ColumnModel();
+        TEST_MODEL_2.setName(TEST_COLUMN_MODEL_NAME_2);
+        TEST_MODEL_2.setColumnType(ColumnType.STRING);
+        TEST_MODEL_2.setMaximumSize(48L);
+        columnListBuilder.add(TEST_MODEL_2);
+
+        TEST_COLUMN_LIST = columnListBuilder.build();
+
+        List<ColumnDefinition> TEST_COLUMN_DEFINITIONS;
+
+        ImmutableList.Builder<ColumnDefinition> columnDefinitionBuilder = ImmutableList.builder();
+
+        ColumnDefinition TEST_DEFINITION_1 = new ColumnDefinition();
+        TEST_DEFINITION_1.setName(TEST_COLUMN_MODEL_NAME_1);
+        TEST_DEFINITION_1.setColumnType(ColumnType.STRING);
+        TEST_DEFINITION_1.setMaximumSize(36L);
+        columnDefinitionBuilder.add(TEST_DEFINITION_1);
+
+        ColumnDefinition TEST_DEFINITION_2 = new ColumnDefinition();
+        TEST_DEFINITION_2.setName(TEST_COLUMN_MODEL_NAME_2);
+        TEST_DEFINITION_2.setColumnType(ColumnType.STRING);
+        TEST_DEFINITION_2.setMaximumSize(48L);
+        columnDefinitionBuilder.add(TEST_DEFINITION_2);
+
+        TEST_COLUMN_DEFINITIONS = columnDefinitionBuilder.build();
+
+        assertEquals(BridgeExporterUtil.convertToColumnList(TEST_COLUMN_DEFINITIONS), TEST_COLUMN_LIST);
+    }
+
+    @Test
+    public void canGetRowValuesFromRecordBasedOnColumnDefinition() {
+        final String TEST_STRING_NAME = "test_string";
+        final String TEST_STRING_SET_NAME = "test_string_set";
+        final String TEST_DATE_NAME = "test_date";
+        final String TEST_EXPORTER_DATE_NAME = "test_exporter_date";
+        final String TEST_SANITIZE = "test_sanitize";
+
+        // setup mock
+        ExportTask mockTask = mock(ExportTask.class);
+        when(mockTask.getExporterDate()).thenReturn(TEST_LOCAL_DATE);
+
+        // create mock record
+        Item testRecord = new Item();
+        testRecord.withString(TEST_STRING_NAME, "test_string_value");
+        testRecord.withStringSet(TEST_STRING_SET_NAME, new String[]{"test_string_set_value_1", "test_string_set_value_2"});
+        testRecord.withLong(TEST_DATE_NAME, 1484181511);
+        testRecord.withString(TEST_EXPORTER_DATE_NAME, "test_exporter_date_value");
+        testRecord.with(TEST_SANITIZE, "test_sanitize_value");
```

Comment:
The test_sanitize_value should have some HTML in it or something to verify we're properly calling the sanitization code.